pom.xml

```diff
@@ -126,6 +128,11 @@
                 <artifactId>fest-assert</artifactId>
                 <version>${version.fest-assert}</version>
             </dependency>
+            <dependency>
+                <groupId>org.assertj</groupId>
+                <artifactId>assertj-core</artifactId>
```

Comment:
`fest-assert` - this is obsolete, assertj is a successor. They also provide automatic conversion from fest to assertj. If we introduce assertj it's a good time to clean it up